dev/io.openliberty.mcp.internal/resources/io/openliberty/mcp/internal/CWMCM.nlsprops

```diff
@@ -0,0 +1,153 @@
+CWMCM0006E.duplicate.special.arguments=CWMCM0006E: The {0} MCP Tool has more than one parameter with type {1}. There may only be one {1} parameter for each Tool method.
+CWMCM0006E.duplicate.special.arguments.explanation=One or more special arguments occured multiples times, only one is expected.
+CWMCM0006E.duplicate.special.arguments.useraction=Remove the extra instances.
+
+# Used for server logging
+# {0} = The fully qualified method name
+# {1} = The parameter type simple name
+CWMCM0007E.invalid.arguments=CWMCM0007E: The {0} MCP Tool has a parameter of type {1} which is not a recognised special argument type and does not have a `@ToolArg` annotation.
+CWMCM0007E.invalid.arguments.explanation=One or more parameters were not annotated as tool arguments and are not recognised special argument types.
+CWMCM0007E.invalid.arguments.useraction=Please check if you have the correct class imported for your argument, or you meant to include @ToolArg.
+
+# Used as part of a response to a client
+# {0} = Request id
+CWMCM0008E.invalid.request.params=CWMCM0008E: A request with ID {0} is already in progress.
+CWMCM0008E.invalid.request.params.explanation= A request with the same id is still in progress. Requests from the same client must have unique IDs.
+CWMCM0008E.invalid.request.params.useraction=Use a unique ID for every request
+
+# Used as part of a response to a client
+# do not translate GET, POST
+CWMCM0009I.get.disallowed=CWMCM0009I: GET requests are not supported.
+CWMCM0009I.get.disallowed.explanation=A GET request was sent but this functionality is not supported.
+CWMCM0009I.get.disallowed.useraction=Use a supported HTTP request i.e POST
+
+# Used for server logging
+# {0} = tool name
+# {1} = exception details
+CWMCM0010E.internal.server.error.detailed=CWMCM0010E: The {0} tool method threw an unexpected exception. The exception was {1}
+CWMCM0010E.internal.server.error.detailed.explanation=The tool method threw an unexpected exception.
+CWMCM0010E.internal.server.error.detailed.useraction=Review the server message.log and FFDC logs to identify the problem.
+
+# Used as part of a response to a client
+CWMCM0011E.internal.server.error=CWMCM0011E: An internal server error occurred while running the tool.
+CWMCM0011E.internal.server.error.explanation=The server was not able to complete the tool call request.
+CWMCM0011E.internal.server.error.useraction=Check that the request is correct. More information about the failure may be found in the server messages.log.
+
+# Used for server logging
+# do not translate bean
+# {0} = bean error details
+# {1} = tool name
+CWMCM0012E.bean.release.fail=CWMCM0012E: An error occurred when releasing the CDI bean after calling the {1} tool. The error is: {0}
+CWMCM0012E.bean.release.fail.explanation=When calling a tool method, an instance of the CDI bean which declares the method is acquired before calling the method and released afterwards. An error occurred when releasing the bean instance.
```

Comment:
Change "which" to "that" unless the change will make the sentence technically incorrect. ("which" either needs a comma after it or it needs to be changed to "that")

Reply:
"that" is correct here, thanks.